Libplanet.Net/DifferentAppProtocolVersionException.cs

```diff
@@ -47,13 +68,28 @@ internal sealed class DifferentAppProtocolVersionException : Exception
         /// <summary>
         /// The protocol version of the current <see cref="Swarm{T}"/>.
         /// </summary>
-        public AppProtocolVersion ExpectedVersion { get; }
+        public AppProtocolVersion Apv { get; }
```

Comment:
`ExpectedVersion` / `ActualVersion` pair is more natual for me than `Apv` / `PeerApv`. 🤔 

@planetarium/libplanet Do you have idea about it?

Reply:
Then what about `ExpectedApv` / `ActualApv`?

Reply:
I agree, as an `Exception`, yes, `Expected` / `Actual` is probably more appropriate. On the other hand, this particular `Exception` carries lot more info than it really should just in order to be used as "context" in certain cases. 😐

Reply:
Eh, sure why not. On second thought, I guess `Expected` / `Actual` might be better.

Reply:
See da61eb5d748fdcebbbe710ee7d8b46dc2d8cf2b5. Even after changing, either way seems a bit weird tho. 🙄